cypress/support/functions.ts

```diff
@@ -168,3 +168,53 @@ Cypress.Commands.add('deleteNamespace', ({namespace, appName}) => {
     cy.contains(appName).should('not.exist');
   }
 });
+
+// Epinio installation functions
+
+// Add the Epinio Helm repo
+Cypress.Commands.add('addHelmRepo', ({repoName, repoUrl}) => {
+  cy.get('.nav').contains('Apps & Marketplace').click();
+  cy.get('.nav').contains('Repositories').click();
+
+  // Make sure we are in Repositories page and we can see the Create button
+  cy.get('h1').contains('Repositories');
+  cy.contains('Create').should('be.visible');
+
+  cy.clickButton('Create');
+  cy.contains('Repository: Create').should('be.visible');
+  cy.typeValue('Name', repoName);
+  cy.typeValue('Index URL', repoUrl);
+  cy.clickButton('Create');
+});
+
+// Install Epinio via Helm
+Cypress.Commands.add('epinioInstall', ({domain, cors}) => {
+    cy.get('.nav').contains('Apps & Marketplace').click();
+    cy.get('.nav').contains('Charts').click();
+    cy.contains('epinio-installer').click();
+    cy.contains('Charts: epinio-installer').should('be.visible');
+    cy.clickButton('Install');
+
+    // Namespace where installation will happen
+    cy.typeValue('Name', 'mynamespace');
+    cy.clickButton('Next');
+    
+    // Configure custom domain if asked
+    if (domain == true) {
```

Comment:
`== true` is not needed, `if (domain) {` is enough. But if you prefer to have it to have something more readable then I'm fine. We will just have to do the same on all code.